config/core.extension.yml

```diff
@@ -16,6 +16,7 @@ module:
   contextual: 0
   datetime: 0
   dblog: 0
+  devel: 0
```

Comment:
yeah we won't want devel turned on on production either likely. May be a good time to implement config ignore and config split.